app/src/main/res/layout/activity_help_parameters.xml

```diff
@@ -0,0 +1,183 @@
+        android:checked="false"
+        android:filterTouchesWhenObscured="false"
+        android:textOff="@string/first_aid_kit"
+        android:textOn="@string/first_aid_kit"
+        app:layout_constraintBottom_toTopOf="@+id/help_params_call_button"
+        app:layout_constraintEnd_toStartOf="@+id/selectMedsButton5"
+        app:layout_constraintStart_toEndOf="@+id/selectMedsButton3"
+        app:layout_constraintTop_toBottomOf="@+id/selectMedsButton1" />
+
+    <ToggleButton
+        android:id="@+id/selectMedsButton3"
+        android:layout_width="110dp"
+        android:layout_height="110dp"
+        android:layout_marginStart="25dp"
+        android:layout_marginTop="2dp"
+        android:layout_marginEnd="2dp"
+        android:layout_marginBottom="60dp"
+        android:background="@drawable/select_meds_button"
+        android:checked="false"
+        android:textOff="@string/cpr"
+        android:textOn="@string/cpr"
+        app:layout_constraintBottom_toTopOf="@+id/help_params_call_button"
+        app:layout_constraintEnd_toStartOf="@+id/selectMedsButton4"
+        app:layout_constraintHorizontal_bias="0.848"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/selectMedsButton0" />
+
+    <ToggleButton
+        android:id="@+id/selectMedsButton0"
+        android:layout_width="110dp"
+        android:layout_height="110dp"
+        android:layout_marginStart="25dp"
+        android:layout_marginTop="224dp"
+        android:layout_marginEnd="2dp"
+        android:layout_marginBottom="2dp"
+        android:background="@drawable/select_meds_button"
+        android:checked="false"
+        android:textOff="@string/epipen"
+        android:textOn="@string/epipen"
+        app:layout_constraintBottom_toTopOf="@+id/selectMedsButton3"
+        app:layout_constraintEnd_toStartOf="@+id/selectMedsButton1"
+        app:layout_constraintHorizontal_bias="0.7"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.787" />
+
+    <ToggleButton
+        android:id="@+id/selectMedsButton5"
+        android:layout_width="110dp"
+        android:layout_height="110dp"
+        android:layout_marginStart="2dp"
+        android:layout_marginTop="2dp"
+        android:layout_marginEnd="25dp"
+        android:layout_marginBottom="60dp"
+        android:background="@drawable/select_meds_button"
+        android:checked="false"
+        android:textOff="@string/med_pro"
+        android:textOn="@string/med_pro"
+        app:layout_constraintBottom_toTopOf="@+id/help_params_call_button"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/selectMedsButton4"
+        app:layout_constraintTop_toBottomOf="@+id/selectMedsButton2" />
+
+    <ToggleButton
+        android:id="@+id/selectMedsButton2"
+        android:layout_width="110dp"
```

Comment:
I suggest adding these dimensions to a separate dimen.xml file so that it is easier to manage and change their respective values later on.

Reply:
Nice catch, this will also come in handy if we change this to dynamic generation later on